recipes/zeromq/all/conanfile.py

```diff
@@ -0,0 +1,111 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class ZeroMQConan(ConanFile):
+    name = "zeromq"
+    homepage = "https://github.com/zeromq/libzmq"
+    description = "ZeroMQ is a community of projects focused on decentralized messaging and computing"
+    topics = ("conan", "zmq", "libzmq", "message-queue", "asynchronous")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "LGPL-3.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "encryption": [None, "libsodium", "tweetnacl"],
```

Comment:
We can migrate migrate https://github.com/conan-community/conan-tweetnacl and add in the future

Reply:
I just tried and it appears zeromq has modified the tweetnacl sources: it includes a implementation of randombytes whereas upstream tweetnacl doesn't.

Reply:
Interesting, thanks for checking it.